### ISSUE TITLE
Missing links on Developer cheat sheet

### ISSUE THREAD

**Issue by Mhitchins** [nhsconnect/gpconnect#408]
[https://developer.nhs.uk/apis/gpconnect-1-1-0/development_deliverables.html](https://developer.nhs.uk/apis/gpconnect-1-1-0/development_deliverables.html)

The first four links in the Explore column in Development assets/cheat sheet don't work. I'm guessing the target folder (gpconnect-fhir/tree/develop/) is no longer there. Anyone know where these files are now?

Also, I presume Structure Definitions is duplicated.

**Comment by briandiggle** [comment]
These like seem date back to a time before FHIR resources were stored centrally on national FHIR reference server https://fhir.nhs.uk/. 
Linking there has some value, but will present the structure definitions etc used by all NHSD programs, not just GP Connect.

We have moved to listing capability specific FHIR details under each capability section. E.g.  https://developer.nhs.uk/apis/gpconnect-1-1-0/datalibraryfoundation.html lists those which pertain to Foundations.

BTW I have added an additional link in this page to the Java code examples in the develop branch - see http://gpconnect.netlify.com/development_deliverables.html
